examples/usecase-todo/package.json

```diff
@@ -7,16 +7,15 @@
     "dev": "keystone dev",
     "start": "keystone start",
     "build": "keystone build",
-    "postinstall": "keystone postinstall",
```

Comment:
> add the schema files to .gitignore so that they are always recreated on the respective machine, using the relevant prisma version from keystone

The downside to this, is we lose the nature of these examples capturing changes to schemas in situations of higher complexity than our unit tests can easily account for

Reply:
Could you elaborate? Doesn't generating them on the fly ensure that they're always in the correct "condition" and therefore true to what we're testing?

Unless I'm missing something, it definitely seems it solves for a couple of issues, minimises management overhead - the only caveat being that users can't browse the files until they've run dev or build.

Reply:
@kennedybaird when I'm developing new features or fixing bugs, changes in our the schemas generated by our examples are quickly detected from the CI errors produced - this has been a highly reliable canary in the coal-mine that what I am changing has behaviour impacts beyond the limited scope of the QA & testing I am applying.

Our unit tests often capture this change in behavior too, but, the expanded scope of the examples have captured nuanced situations which I don't often think about.

Reply:
Another way of framing this, our `examples/` are acting as snapshot tests

Reply:
Copy that - it makes sense. It's given me some other thoughts that I've noted, which I'll put somewhere else.